peribolos/knative-OWNERS_ALIASES

```diff
@@ -89,94 +84,53 @@ aliases:
   - knative-prow-robot
   - knative-prow-updater-robot
   - knative-test-reporter-robot
-  networking-reviewers:
-  - JRBANCEL
-  - ZhiminXiang
-  - andrew-su
-  - carlisia
-  - nak3
-  - tcnghia
-  - vagababov
-  - yanweiguo
-  networking-wg-leads: []
-  networking-writers:
-  - JRBANCEL
-  - vagababov
   operations-reviewers:
-  - Cynocracy
   - aliok
   - houshengbo
-  - jcrossley3
   - matzew
   - maximilien
   operations-wg-leads:
   - houshengbo
   operations-writers:
-  - Cynocracy
   - aliok
   - houshengbo
-  - jcrossley3
   - matzew
   - maximilien
-  pkg-configmap-reviewers:
-  - dprotaso
-  - mattmoor
-  - vagababov
-  pkg-configmap-writers:
-  - dprotaso
-  - mattmoor
-  - vagababov
-  pkg-controller-reviewers:
-  - dprotaso
-  - mattmoor
-  - tcnghia
-  - vagababov
-  pkg-controller-writers:
-  - dprotaso
-  - mattmoor
-  - tcnghia
-  - vagababov
   productivity-leads:
   - kvmware
   - upodroid
   productivity-reviewers:
   - evankanderson
   - mgencur
-  - shinigambit
   productivity-wg-leads:
   - kvmware
   - upodroid
   productivity-writers:
   - cardil
-  - chaodaiG
-  - coryrc
   - kvmware
   - psschwei
   - upodroid
   security-wg-leads:
   - evankanderson
   security-writers:
   - evankanderson
-  serving-observability-reviewers:
-  - skonto
-  - yanweiguo
-  serving-observability-writers:
-  - yanweiguo
+  serving-approvers:
```

Comment:
what's the difference between serving-approvers and serving-writers?

Reply:
writers includes approvers and wg leads